applications/osb-portal/src/components/header/Banner.tsx

```diff
@@ -58,9 +58,9 @@ export const Banner = (props: any) => {
               <Button variant="outlined">Take the tour</Button>
             }
             {user === null ? (
-              <Button variant="outlined" onClick={handleSignup}>Sign up</Button>
+              <Button id='signup' variant="outlined" onClick={handleSignup}>Sign up</Button>
```

Comment:
The naming should be consistent and indicate more about the object and functionality.

`signup-button` here would be appropriate also for consistency with other -button ids